docs/modules/concepts/pages/glossary.adoc

```diff
@@ -0,0 +1,31 @@
+= Glossary
+
+xref:mail/messages/imf.adoc[IMF]::
+*Internet Message Format* (IMF) is the base specification that defines the
+syntax of an email message.
+
+IMAP::
+
+JMAP::
+
+xref:mail/messages/mime.adoc[MIME]::
+*Multipurpose Internet Mail Extension* (MIME) is a specification that extends
+IMF, defining most of the "advanced" features of email.
+
+MTA::
+A *Mail Transfer Agent* (MTA) is a host that is capable of providing a mail
+transport service compliant with the SMTP protocol.
+
+MUA::
+A *Mail User Agent* (MUA) is either an agent that hands over a message to an initial
```

Comment:
Well, I think that it's not clear from that description it's what people uses, it's the visible part of the email infrastructure

Reply:
Agree. I'm not sure where to link to. The only description in the text so far is in the SMTP section.

The idea of the glossary is to provide only a very short-basic description with a link to a more in-depth description. I need to figure out if MUA is "important" enough to warrant its own section.